incubator/hnc/cmd/manager/main.go

```diff
@@ -81,6 +83,7 @@ func init() {
 func main() {
 	setupLog.Info("Parsing flags")
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
```

Comment:
shall we name it to "healthz-addr" to make it consistent with metrics-addr?
It will make it easier to read and use.